build.gradle

```diff
@@ -104,5 +104,5 @@ buildScan {
 }
 
 ext {
-    fluxCVersion = 'c17892263bb2be9cdeca41cdd0156179127f2cdf'
+    fluxCVersion = 'b6f8aa35c5e5782f126cb4e40966d5484ad1d494'
```

Comment:
Is this all I need to do for FluxC changes?

Reply:
Yes, ideally we should merge the FluxC PR and update the commit hash here with the one after merge, to make sure it's up to date with FluxC's develop. 
SideNote: In cases where changes in FluxC are breaking changes, we usually review both PRs and merge them right after each other so we limit the time during which FluxC's develop is not compatible with WPAndroid develop. However, there are no breaking changes in this case.